setup.py

```diff
@@ -535,15 +536,24 @@ def pxd(name):
         'depends': _pxi_dep['sparse']},
     '_libs.tslib': {
         'pyxfile': '_libs/tslib',
-        'pxdfiles': ['_libs/src/util'],
+        'pxdfiles': ['_libs/src/util',
+                     '_libs/tslibs/conversion',
+                     '_libs/tslibs/timedeltas',
+                     '_libs/tslibs/timezones',
+                     '_libs/tslibs/nattype'],
         'depends': tseries_depends,
         'sources': np_datetime_sources},
     '_libs.tslibs.conversion': {
         'pyxfile': '_libs/tslibs/conversion',
+        'pxdfiles': ['_libs/src/util',
+                     '_libs/tslibs/timezones',
+                     '_libs/tslibs/np_datetime',
+                     '_libs/tslibs/timedeltas'],
         'depends': tseries_depends,
         'sources': np_datetime_sources},
     '_libs.tslibs.fields': {
         'pyxfile': '_libs/tslibs/fields',
+        'pxdfiles': ['_libs/src/util'],
```

Comment:
The functions defined in util are evidently copy/pasted in the generated C code here even without this entry being present.  See previous question about "what is this intended to do?"

Reply:
I think these are orthogonal, e.g. the dependency based compilation and the actual c code generation.